build/dependencies.props

```diff
@@ -18,8 +18,8 @@
   <!-- Partner Versions-->
   <PropertyGroup>
     <CodeAnalysisVersion>1.3.0</CodeAnalysisVersion>
-    <MicrosoftBuildVersion>15.1.548</MicrosoftBuildVersion>
-    <NewtonSoftVersion>9.0.1</NewtonSoftVersion>
+    <MicrosoftBuildVersion>15.2.0-preview-000047-02</MicrosoftBuildVersion>
+    <NewtonSoftVersion>10.0.1</NewtonSoftVersion>
```

Comment:
JsonNetBsonVersion is a different dependency `Newtonsoft.Json.Bson`
I am assuming for Newtonsoft.Json it should be JsonNetVersion